controllers/comments.controller.js

```diff
@@ -35,3 +36,18 @@ exports.postComment = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeCommentById(comment_id)
+    .then((deletedComment) => {
+      if (deletedComment.rowCount === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: "comment does not exist",
+        });
+      }
+      res.status(204).send();
+    })
+    .catch(next);
```

Comment:
This works, but we should not be handling errors like in the controller file. When we enter the .then(...) here we should assume we are following the happy path and should not have to check rows like this